From a92657ecd2f121f5f42f4882a3405cb58659377f Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Wed, 23 Mar 2022 09:33:20 +0200
Subject: [PATCH] [SW-69581] gaudi3: adjust fw sram base to latest spec

FW SRAM base address changed because of arc1 memory growth

Change-Id: Ib3a1129e5e20e6754ab00246b6a5e728ba1557c8
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.h b/arch/gaudi3/arc/common.h
index 57d6018..df5e5ef 100644
--- a/arch/gaudi3/arc/common.h
+++ b/arch/gaudi3/arc/common.h
@@ -33,7 +33,7 @@
 #define SDRAM_SIZE			(SZ_32M + SZ_16M) /* single die mode */
 #define SDRAM_BASE			(FW_MEM_REGION_BASE - SDRAM_SIZE)
 #define PCIE_FW_SRAM_BASE		(FW_MEM_REGION_BASE + 0x7F00000)
-#define FW_SRAM_BASE			(FW_MEM_REGION_BASE + 0x30E0000)
+#define FW_SRAM_BASE			(FW_MEM_REGION_BASE + 0x3140000)
 #define SP_SRAM_BASE			(FW_MEM_REGION_BASE + 0x3120000)
 #define ARC1_ICCM_BASE			(FW_MEM_REGION_BASE + 0x3040000)
 #define ARC0_ICCM_BASE			(FW_MEM_REGION_BASE + 0x3000000)
